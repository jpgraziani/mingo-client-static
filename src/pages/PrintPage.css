.freeImg-container {
  width: 100%;
  margin: 0 auto;
  display: flex;
  flex-direction: column;
  background-color: var(--deep-rose);
  align-items: center;
  justify-content: center;
  padding: 2.5rem 0;
}

.img-download {
  width: 80%;
  text-align: center;
  border: 4px solid var(--deep-rose);
  background-color: var(--white);
  padding: 1.5rem;
  transition: all 0.3s ease-out;
}

.img-download:hover {
  transform: translateY(-5px) scale(1) translateZ(0);
}

.img-cover {
  box-sizing: border-box;
}

.download-btn a:link {
  display: inline-block;
	margin-top: 1rem;
	margin-right: 1rem;
	padding: .9rem 1rem;
	text-decoration: none;
	font-size: 1.5rem;
	letter-spacing: 1px;
	font-weight: 600;
  border: 2px solid var(--deep-rose);
  color: var(--deep-rose);
}

.download-btn a:visited {
	border: 2px solid var(--deep-rose);
	color: var(--deep-rose);
}

.download-btn a:hover{
	border: 2px solid var(--deep-rose);
	background-color: var(--deep-rose);
	color: var(--white);
}