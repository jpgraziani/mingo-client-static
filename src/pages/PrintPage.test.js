import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PrintPage from './PrintsPage'

describe(`PrintPage component`, () => {
  it('renders an PrintPage by default', () => {
    const wrapper = shallow(<PrintPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})