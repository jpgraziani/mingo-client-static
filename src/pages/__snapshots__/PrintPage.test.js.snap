// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrintPage component renders an PrintPage by default 1`] = `
<section
  className="freeImg-container"
>
  <div
    className="img-download"
    key="0"
  >
    <img
      alt={0}
      className="img-cover"
      key="0"
      src="deep-diver-color.png"
    />
    <div
      className="download-btn"
    >
      <a
        download={true}
        href="deep-diver-color.png"
      >
        download
      </a>
    </div>
  </div>
  <div
    className="img-download"
    key="1"
  >
    <img
      alt={1}
      className="img-cover"
      key="1"
      src="fisherman.png"
    />
    <div
      className="download-btn"
    >
      <a
        download={true}
        href="fisherman.png"
      >
        download
      </a>
    </div>
  </div>
  <div
    className="img-download"
    key="2"
  >
    <img
      alt={2}
      className="img-cover"
      key="2"
      src="viking.png"
    />
    <div
      className="download-btn"
    >
      <a
        download={true}
        href="viking.png"
      >
        download
      </a>
    </div>
  </div>
</section>
`;
